java/sage/Agent.java

```diff
@@ -465,7 +474,7 @@ void unbully(int weakerID)
       for (Show show : shows) {
         Airing[] airings = wiz.getAirings(show, 0);
         for(Airing a : airings) {
-          if (followsTrend(a, true, sbCache, true))
+          if (followsTrend(a, true, sbCache, true, true))
```

Comment:
This should pass through ignoreDisabled as the last value (I understand it'll have the same functionality, but it just looks more correct).